src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/DependencyResolution/ConfigurationRegistry.cs

```diff
@@ -20,6 +20,7 @@ public ConfigurationRegistry()
             AddConfiguration<ProviderFeaturesConfiguration>(ProviderCommitmentsConfigurationKeys.FeaturesConfiguration);
             AddConfiguration<ZenDeskConfiguration>(ProviderCommitmentsConfigurationKeys.ZenDeskConfiguration);
             AddConfiguration<DataProtectionConnectionStrings>(ProviderCommitmentsConfigurationKeys.DataProtectionConnectionStrings);
+            AddConfiguration<CsvConfiguration>(ProviderCommitmentsConfigurationKeys.CsvConfiguration);
```

Comment:
Would this be better name BulkUploadFileValidationConfiguration or something similarly specific?

Reply:
changed